packages/generator-electrode/generators/app/templates/server/index.js

```diff
@@ -32,4 +32,13 @@ supports.babelRegister({
 supports.cssModuleHook({
   generateScopedName: "[name]__[local]___[hash:base64:5]"
 });
-require<% if (isHapi) { %>("electrode-server")(config, [staticPathsDecor()])<% } else { %>("./express-server")(config)<% } %>;
+require<% if (isHapi) {
+  %>("electrode-server")(config, [staticPathsDecor()]).then((server) => {
```

Comment:
note that electrode-server is only used when developing/testing the reporter.  When the reporter is actually being used by the user, it's part of webpack's dev server and there's no electrode server.